src/component/selection/setDraftEditorSelection.js

```diff
@@ -303,9 +303,14 @@ function addFocusToSelection(
     // Additionally, clone the selection range. IE11 throws an
     // InvalidStateError when attempting to access selection properties
     // after the range is detached.
-    var range = selection.getRangeAt(0);
-    range.setEnd(node, offset);
-    selection.addRange(range.cloneRange());
+    //
+    // Added rangeCount check
+    // https://stackoverflow.com/questions/22935320/uncaught-indexsizeerror-failed-to-execute-getrangeat-on-selection-0-is-not
+    if (selection.rangeCount > 0) {
```

Comment:
I've run into this before, legit error! Thanks for the pr! I'll bring it up and hopefully we can merge this in soonish